.gitignore

```diff
@@ -7,3 +7,4 @@ coverage
 .settings
 composer.lock
 docs-api
+.idea
```

Comment:
You should add this to a global ignore: http://stackoverflow.com/questions/7335420/global-git-ignore